extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(brazil) {
+  let ninetyNine=(brazil*99)/100;
```

Comment:
Let's think about how we can make this code even clearer.
You've created a variable to save the value after you've taken away 1%. But what about the expression - `ninetyNine * 5.7` - what does this represent ? Could you store this expression in a variable and give it a name so it's clearer what exactly you're calculating.